mysql/tests/conftest.py

```diff
@@ -143,6 +144,17 @@ def _wait_for_it_script():
     return os.path.abspath(script)
 
 
+def _mysql_conf_path():
+    if MYSQL_FLAVOR == 'mysql':
+        filename = 'mysql.conf'
+    else:
+        assert MYSQL_FLAVOR == 'mariadb'
+        filename = 'mariadb.conf'
```

Comment:
(nit) This assert is a bit odd. Maybe something like this is more appropriate.

```

    if MYSQL_FLAVOR == 'mysql':
        filename = 'mysql.conf'
    elif MYSQL_FLAVOR == 'mariadb':
        filename = 'mariadb.conf'
    else:
        raise Exception("Invalid MYSQL_FLAVOR: {}".format(MYSQL_FLAVOR))
```

Reply:
I like using asserts for things that should hold true but do not depend user input (e.g. developer sanity checks). This is an example of it since we implicitly expect to only ever have either `mysql` or `mariadb` as flavors (at least for now).

But your suggestion works great too — I'll push the change. :-)